common/assets/scss/components/_whats-new-banner.scss

```diff
@@ -0,0 +1,17 @@
+.whats-new-banner {
+	width: 100% ; 
+	height: 279px;
+	left: 0px;
+	top: 178px;
```

Comment:
Is there a way we can make these numbers more dynamic? I'm a little nervous of what might happen if a user zooms in or has custom font sizes, whether the text might get cut off. cc @ElizabethGoddard

Reply:
Good shout, Ill do this now.